trace-mini-agent/src/mini_agent.rs

```diff
@@ -55,11 +56,11 @@ impl MiniAgent {
 
         let server = server_builder.serve(make_svc);
 
-        info!("Mini Agent listening on port {}", MINI_AGENT_PORT);
+        info!("Mini Agent started: listening on port {MINI_AGENT_PORT}");
 
         // start hyper http server
         if let Err(e) = server.await {
-            error!("Server error: {}", e);
+            error!("Server error: {e}");
         }
 
         Ok(())
```

Comment:
If the server fails, we still return `Ok(())`?  Seems wrong although I don't know why it would fail at this spot.

Reply:
you're right, I changed this to return the server error